### PR TITLE
Better error reporting

### PR DESCRIPTION
**Pull request by nraychaudhuri**

### CODE REVIEW

**Comment by skyluc** [comment]
LGTM (for the last commit)

**Comment by skonto** [comment]
Cool stuff with console withOut ;)

File: mesos-docker/run/mit-application.conf.template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+#Used to copy the application jar file for cluster mode tests
+mit.hdfs.uri = "replace_with_hdfs_uri"
+
+#The IP address of the host as seen from docker instance. This ip address is used to connect to runner
+#from docker instance.
+mit.docker.host.ip = "replace_with_docker_host_ip" 
+
+#path of mesos slaves to find spark executor. This should be set when you create docker based mesos cluster
+mit.spark.executor.tgz.location = "replace_with_executor_tgz_location"
```
Comment (dragos) [inline mesos-docker/run/mit-application.conf.template]: Is this the same as `spark.executor.uri`? Let's use that, it's probably clearer what it means.

**Comment by skonto** [comment]
Update Readme.md file for mesos-docker pls...for example now we have 2 slaves by default...

"The most simple run is:

./run.sh

which creates a simple cluster with 1 mesos master and 1 mesos slave with the latest spark version installed. Another common usage is:"

**Comment by nraychaudhuri** [comment]
great feedback. Thx. I will make necessary changes

**Comment by nraychaudhuri** [comment]
move the changes to #8 PR. Closing this right now

**Pull request closed without merging**